java/java-ranger-regression/printtokens_eqchk/impl/token_stream.java

```diff
@@ -7,26 +7,26 @@
 public class token_stream {
 
 	private PushbackReader reader = null;
-	
-	public token_stream(Reader _reader){
+
+	public token_stream(Reader _reader) {
 		reader = new PushbackReader(_reader, 10);
 	}
-	
+
```

Comment:
Formatting looks still a bit off. (spaces vs tabs)

Reply:
In my opinion, formatting should be enforced by a linter in the CI. This might save us some time. I created #1170 to discuss this further.